src/hooks/useSearchSubmit.ts

```diff
@@ -2,6 +2,7 @@ import { useCallback } from 'react'
 import { useHistory } from 'react-router-dom'
 import { isAnAddressAccount } from 'utils'
 import { usePathPrefix } from 'state/network'
+import { web3 } from 'apps/gp-v1/api'
```

Comment:
We shouldn't import web3 from the old gp-v1 (which is the old Gnosis Protocol, AKA Mesa)

Reply:
A hint of another place to get it 
![image](https://user-images.githubusercontent.com/2352112/137523851-c51b4181-0223-4282-a14f-7fb277308f49.png)